### ISSUE TITLE
Unreal Engine 4 Endpoint CSRF Error

### ISSUE THREAD

**Issue by randrewy** [getsentry/onpremise#995]
### Version Information

Version: *21.6.1*

### Steps to Reproduce

1. Get latest release
2. run `install.sh` on WSL
3. run `docker-compose up` on Windows
4. create new project `unreal`
5. configure CrashReporter with 'Unreal Engine 4 Endpoint'
6. attempt to send crash

### Expected Result
New crash event occures in Sentry dashboard

### Actual Result
Crash fails to send due to CSRF error

### Logs
```
nginx_1                                     | 2021/06/16 16:40:06 [warn] 7#7: *125 a client request body is buffered to a temporary file /var/cache/nginx/client_temp/0000000001, client: 172.18.0.1, server: , request: "POST /api/2/unreal/df00d55472eb41edb61f6b2ed5108080?AppID=CrashReporter&AppVersion=5.0.0-16433597%2B%2B%2BUE5%2BRelease-5.0-EarlyAccess&AppEnvironment=Release&UploadType=crashreports&UserID=235256824fca9869f9b3908ef1c3042f%7Cf7959b6088b8447abe94b9c7f56e7eb1%7C0c548ec3-76d3-48e8-84ff-377674b1f8f8 HTTP/1.1", host: "127.0.0.1:9000"
web_1                                       | 16:40:06 [WARNING] django.security.csrf: Forbidden (CSRF cookie not set.): /api/2/unreal/df00d55472eb41edb61f6b2ed5108080 (status_code=403 request=<WSGIRequest: POST '/api/2/unreal/df00d55472eb41edb61f6b2ed5108080?AppID=CrashReporter&AppVersion=5.0.0-16433597%2B%2B%2BUE5%2BRelease-5.0-EarlyAccess&AppEnvironment=Release&UploadType=crashreports&UserID=235256824fca9869f9b3908ef1c3042f%7Cf7959b6088b8447abe94b9c7f56e7eb1%7C0c548ec3-76d3-48e8-84ff-377674b1f8f8'>)
nginx_1                                     | 172.18.0.1 - - [16/Jun/2021:16:40:06 +0000] "POST /api/2/unreal/df00d55472eb41edb61f6b2ed5108080?AppID=CrashReporter&AppVersion=5.0.0-16433597%2B%2B%2BUE5%2BRelease-5.0-EarlyAccess&AppEnvironment=Release&UploadType=crashreports&UserID=235256824fca9869f9b3908ef1c3042f%7Cf7959b6088b8447abe94b9c7f56e7eb1%7C0c548ec3-76d3-48e8-84ff-377674b1f8f8 HTTP/1.1" 403 2652 "-" "CrashReportClient/++UE5+Release-5.0-EarlyAccess-CL-16433597 Windows/10.0.19033.1.256.64bit" "-"
```

What you saw along the way, e.g.:

- I'm unable to run `docker-compose up` on WSL due to some errors, though it seems irrelevant to ussue
- Posting Minidump directly to minidump endpoint with curl worked perfectly fine

**Comment by BYK** [comment]
>I'm unable to run docker-compose up on WSL due to some errors, though it seems irrelevant to ussue

This sounds relevant as you are hitting the Sentry web instance whereas your request to `/api/2/...` should be handled by Relay.

**Comment by randrewy** [comment]
Hello BYK!

I've tried to use WSL again and it started successfully, but there was the same csrf error.
I've pruned docker system & volumes several times ang reinstalled everything from scratch and it looks like sometimes `docker-compose up` fails due to File not found error (sorry log is lost)

Nevertheless i've found the problem:
```
http://127.0.0.1:9000/api/2/minidump/?sentry_key=df00d55472eb41edb61f6b2ed5108080
```
this URL routes to `sentry` upstream for some reason, but if final `/` is added it starts to route to `relay`

so from my side fix is simple, just use URL with final `/`
```
http://127.0.0.1:9000/api/2/minidump/?sentry_key=df00d55472eb41edb61f6b2ed5108080/
```

Thhe issue can be closed I think. Fell free to ask any additianal info though
